### PR TITLE
feat(Border Radius): Fixing Atoms' border radii

### PR DESCRIPTION
**Pull request by LinKCoding**
### Overview
Addressing the atoms first, fixing border radii
See notion for more details: https://www.notion.so/codecademy/Border-Radius-Tracker-55c1c9e9ab49480995d314ac3fb784ff

### PR Checklist

- [ ] Related to designs:
- [ ] Related to a mixture of JIRA tickets: [GM-592](https://skillsoftdev.atlassian.net/browse/GM-592), [GM-892](https://skillsoftdev.atlassian.net/browse/GM-829), [GM-828](https://skillsoftdev.atlassian.net/browse/GM-828), [GM-830](https://skillsoftdev.atlassian.net/browse/GM-830)
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. Visit the notion page to see a list of changed components, go to their relative paths and check that things render the same and work the same way. 

[GM-592]: https://skillsoftdev.atlassian.net/browse/GM-592?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GM-828]: https://skillsoftdev.atlassian.net/browse/GM-828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GM-830]: https://skillsoftdev.atlassian.net/browse/GM-830?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/gamut/src/Tag/elements.tsx [hunk 1]
```diff
@@ -54,7 +53,7 @@ export const DismissButton = styled(IconButton)(
       ...iconButtonOverrides,
       color: 'background',
       border: 'none',
-      borderRadius: `0 ${tagBorderRadius} ${tagBorderRadius} 0`,
+      borderRadius: `0 ${theme.borderRadii.m} ${theme.borderRadii.m} 0` as any,
```
Comment (LinKCoding) [inline packages/gamut/src/Tag/elements.tsx:56]: getting a TS error when setting as const :\

Comment (dreamwasp) [reply packages/gamut/src/Tag/elements.tsx:56]: maybe try declaring outside the css object and passing it in? i;m chill with this either way

**Review by dreamwasp — COMMENTED** [review]
left some code-only review comments, gonna test locally in a bit

File: packages/gamut/src/Tag/elements.tsx (hunk 1 above)
Comment (LinKCoding) [reply packages/gamut/src/Tag/elements.tsx:56]: Didn't seem to work :\ 
image

gonna just keep it as is

**Pull request merged**